### DISCUSSION TITLE
FAQ: Lossless / Hi-Fi Support (STOP MAKING DUPLICATE REQUESTS)

### DISCUSSION THREAD

**Discussion by jacobfromNM** [ciderapp/Cider#889] [Feature Request]
Hi there - thank you all for your great work on this application! The only minor feature I'm missing from the previous app is the high bit-rateplayback options. Otherwise - the app is fantastic!

**Comment by Amaru8** [comment]
We are glad you like the app. 🙂 Lossless playback is currently not supported in Cider. Apple's MusicKit Framework does have lossless support, however, decryption of this audio is not supported.

**Reply by jacobfromNM** [reply]
Understood - I really appreciate the prompt reply! The previous player supported playback up to 990kbps - would it be possible to have this implemented here?

**Reply by GamingLiamStudios** [reply]
iirc, 990kbps was not an officially supported feature with MusicKit and is more of a hack. It was also not supported by every song.
